packages/devextreme-cli/src/templates/react/application/src/layouts/side-nav-outer-toolbar/side-nav-outer-toolbar.tsx

```diff
@@ -44,7 +44,7 @@ export default function SideNavOuterToolbar({ title, children }<%=#isTypeScript%
         : prevMenuStatus
     );
     return menuStatus === MenuStatus.Closed ? true : false;
-  }, [isLarge]);
+  }, [isLarge, MenuStatus]);
```

Comment:
MenuStatus can't be in dependency array, because it is const
u should use menuStatus here